test/auth/e2e.spec.ts

```diff
@@ -62,4 +72,56 @@ describe('auth', () => {
       await expect(page.locator('#use-auth-result')).toHaveText('Goodbye, world!')
     })
   })
+
+  describe('api-keys', () => {
+    let user
+
+    beforeAll(async () => {
+      url = new AdminUrlUtil(serverURL, apiKeysSlug)
+
+      user = await payload.create({
+        collection: apiKeysSlug,
+        data: {
+          enableAPIKey: true,
+        },
+      })
+    })
+
+    test('should enable api key', async () => {
+      await page.goto(url.create)
+
+      // click enable api key checkbox
+      await page.locator('#field-enableAPIKey').click()
+
+      // assert that the value is set
+      const apiKey = await page.locator('#apiKey').inputValue()
+      expect(apiKey).toBeDefined()
+
+      await saveDocAndAssert(page)
+
```

Comment:
@DanRibbens I've also tried to fetch the user in this line using the API key created in this test, but the API still returns a `null` user. For some reason Auth isn't working as expected here. The following fails:

```typescript
      // use the api key in a fetch to assert that it is enabled
      const response = await fetch(`${apiURL}/${apiKeysSlug}/me`, {
        headers: {
          ...headers,
          Authorization: `${slug} API-Key ${apiKey}`,
        },
      }).then((res) => res.json())

      expect(response.user).not.toBeNull()
      ```